src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -5,6 +5,14 @@
     <Using Include="Sentry" />
   </ItemGroup>
 
+  <ItemGroup>
+    <CompilerVisibleProperty Include="PublishAot" />
+    <CompilerVisibleProperty Include="UseDotNetNativeToolchain" />
+    <CompilerVisibleProperty Include="PublishTrimmed" />
+    <CompilerVisibleProperty Include="DisableSentrySourceGenerator" />
+    <CompilerVisibleProperty Include="Configuration" />
```

Comment:
question: Do we need `"UseDotNetNativeToolchain"` and `"Configuration"`?

Reply:
Yes - there were other stories linked to source generator outside of AOT for UWP/WinUI stuff.  

Configuration could help in the future.  It costs us nothing.